### PR TITLE
Turkish update

### PR DESCRIPTION
**Pull request by OzancanKaratas**

### CODE REVIEW

File: Client/TeamTalkAndroid/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -188,6 +188,7 @@
   <string name="err_retrieve_public_server_list">Kamu sunucusu listesini getirme başarısız</string>
   <string name="err_send_text_message">Metin iletisini gönderme başarısız</string>
   <string name="err_connection">Bağlantı başarısız</string>
+  <string name="err_facebooklogin">Facebook oturumu başarısız</string>
```
Comment (Oreonan) [inline Client/TeamTalkAndroid/src/main/res/values-tr/strings.xml:191]: Can you remove this line, Facebook login has been removed.

File: Client/TeamTalkAndroid/src/main/res/values-tr/strings.xml [hunk 2]
```diff
@@ -205,6 +206,7 @@
   <string name="pref_title_authentication">Kimlik Doğrulama</string>
   <string name="pref_title_username">Kullanıcı Adı</string>
   <string name="pref_title_password">Parola</string>
+  <string name="pref_title_facebook">Facebook Oturumu</string>
```
Comment (Oreonan) [inline Client/TeamTalkAndroid/src/main/res/values-tr/strings.xml:209]: Can you remove this line, Facebook login has been removed?

File: Client/TeamTalkAndroid/src/main/res/values-tr/strings.xml [hunk 3]
```diff
@@ -322,8 +324,10 @@
   <string name="bwpswlab">Parola:</string>
   <string name="ttversion">TeamTalk v</string>
   <string name="ttdllversion">TeamTalk DLL v</string>
+  <string name="pref_title_facebook_summary">Yalnızca bu sunucu Facebook oturumunu etkinleştirdiyse işaretleyin</string>
```
Comment (Oreonan) [inline Client/TeamTalkAndroid/src/main/res/values-tr/strings.xml:327]: Can you remove this line, Facebook login has been removed?

**Comment by OzancanKaratas** [comment]
@bear101, please update translations for all clients before merging.

**Pull request closed without merging**